rules/ast/ast.go

```diff
@@ -376,6 +376,7 @@ func (node *VectorAggregation) labelsToGroupingKey(labels clientmodel.Metric) ui
 	summer := fnv.New64a()
 	for _, label := range node.groupBy {
 		fmt.Fprint(summer, labels[label])
+		fmt.Fprint(summer, []byte{0})
```

Comment:
(and I guess when we fix the fingerprinting in the client library, we should also use the `SeparatorByte` instead of `0`)